src/wechaty-puppet/schemas/image.go

```diff
@@ -3,8 +3,8 @@ package schemas
 type ImageType uint8
 
 const (
-	Unknown   ImageType = 0
-	Thumbnail           = 1
-	HD                  = 2
-	Artwork             = 3
+	ImageTypeUnknown   ImageType = 0
```

Comment:
Q1: Go does not allow const of the same name and different types in the same package;so I changed the name of the constant, which may differ from the community specification,I wonder if it is appropriate?